### PR TITLE
Skin fixes pt2: ensure font settings are applied, small fixes

### PR DESCRIPTION
**Pull request by ronso0**
* ensure all font styles are applied and no system 'sans serif' has to jump in
* add AutoDJ indicator to the toolbars and Shade mixer, including new tooltip `AutoDJ is active`
* Shade: fix track colors, add to mini decks, use correct knob arcs, add inverted arc for Ducking

### CODE REVIEW

**Comment by ronso0** [comment]
@Holzhaus Please test Deere where you recently noticed the font issue.

**Comment by Holzhaus** [comment]
> @Holzhaus Please test Deere where you recently noticed the font issue.

Look like it's fixed, thanks!

Screenshot from 2020-11-04 23-29-26

**Review by Holzhaus — APPROVED** [review]
LGTM. Also fixes the Tango warnings.

File: res/skins/Tango/topbar.xml [hunk 1]
```diff
@@ -140,7 +140,25 @@ Description:
         <SizePolicy>me,min</SizePolicy>
         <Children>
 
-          <WidgetGroup><Size>10me,1min</Size></WidgetGroup>
+          <StatusLight>
+            <TooltipId>FIXME</TooltipId>
```
Comment (Holzhaus) [inline res/skins/Tango/topbar.xml:144]: Maybe add a TODO to the PR description?

Comment (ronso0) [reply res/skins/Tango/topbar.xml:144]: Right, I actually wanted to fix that in this branch.
Funny thing is, even though it has a custom fixed tooltip the keyboard shortcut is appended:
image

Comment (ronso0) [reply res/skins/Tango/topbar.xml]: Done.
image

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Comment by uklotzde** [comment]
@ronso0 We need your help again to merge 2.3 to master. I am clueless how to resolve the conflicts.

Merge branch '2.3' of git@github.com:mixxxdj/mixxx.git into main

# Conflicts:
#	res/skins/LateNight/style.qss
#	res/skins/Shade/style/style_bg_mixer_panel_main.png

**Pull request merged**